portafly/src/components/shared/index.tsx

```diff
@@ -0,0 +1,14 @@
+// export * from './developer-accounts/DeveloperAccountsTable'
+// export * from './AppLayout'
+// export * from './AppNavExpandable'
+// export * from './AppNavGroup'
+// export * from './AppNavItem'
+export * from './data-list'
+export * from './EmptyState'
+// export * from './LazyRoute'
+// export * from './PrivateRoute'
+export * from './Loading'
+// export * from './NotFound'
+// export * from './SwitchWith404'
+// export * from './util'
+// export * from './Root'
```

Comment:
Any reason of keeping this commented lines?

Reply:
🤯